rest/src/routes/diagnosticRoutes.js

```diff
@@ -39,5 +35,24 @@ module.exports = {
 				next();
 			});
 		});
+
+		server.get('/diagnostic/serverInfo', (req, res, next) => {
```

Comment:
would probably prefer dropping info as /server and /storage seem more consistetn